mep/settings.py

```diff
@@ -12,11 +12,13 @@
 
 import os
 
-# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+# Full filesystem path to the project.
+PROJECT_APP_PATH = os.path.dirname(os.path.abspath(__file__))
+PROJECT_APP = os.path.basename(PROJECT_APP_PATH)
+PROJECT_ROOT = BASE_DIR = os.path.dirname(PROJECT_APP_PATH)
 
 # Default debug to False, override locally
-DEBUG = False
+DEBUG = True
```

Comment:
This seems like a mistake we should fix, since we don't want debug accidentally turned on in production

Reply:
Agreed. Thanks for the spot. The `local_settings` should always force otherwise, but I'd rather have the default be no always.